client/app/locales/locale-de.json

```diff
@@ -23,7 +23,8 @@
     "REDIRECT": "Umleitung zu deiner Gruppe..."
   },
   "TOPBAR": {
-    "LOGOUT": "Ausloggen"
+    "LOGOUT": "Ausloggen",
+    "GREETING": "Moin, {{username}}!"
```

Comment:
yes, we use locale-en.json as source file for all translations. so the workflow is:
write key-value pair in `locale-en.json`, merge it to master, upload it to transifex, translate it, download the resulting `locale-**.json` files

Might put this in the README ;)